package.json

```diff
@@ -7,15 +7,6 @@
   },
   "homepage": "https://cloudscape.design",
   "main": "./index.js",
-  "files": [
-    "bootstrap",
-    "components",
-    "schema",
-    "test-utils",
-    "test-utils-new",
-    "*.js",
-    "*.d.ts"
-  ],
```

Comment:
This is a hard-coded list of directories which breaks every time we change directory structure.

With the internal build process improvements, we do not need this list anymore. Checked with the internal dry-run: 7520468914